test/perf/benchmarks/Engine.Parser.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Management.Automation.Language;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace Engine
+{
+    [BenchmarkCategory(Categories.Engine, Categories.Public)]
+    public class Parsing
+    {
+        [Benchmark]
+        public Ast UsingStatement()
+        {
+            const string script = @"
```

Comment:
Maybe move this to setup? So the benchmarch only parses.

Reply:
This string is a const, so it's evaluated only once.